templates/extra/resources.html

```diff
@@ -125,37 +125,37 @@ <h2 class="extra-padding">Artists Resources</h2>
                         <p>
-                            Before applying for PerDiem, you should read (resources) to get a better understanding of how the music industry works. If you are a veteran and fully understand the music industry. Then go ahead and apply. 
+                            Before applying for PerDiem, you should read (resources) to get a better understanding of how the music industry works. If you are a veteran and fully understand the music industry. Then go ahead and apply.
                         </p>
                     </div>
                     <div id="join-2">
                         <h4>Amount to raise</h4>
                         <p>
-                            You need to find out exactly how much you need to raise to create your project. Make sure you discuss all of the costs with your band members, producer, studio, etc… Some unknown costs can come up that are not expected. This can include mixing and master, extra recording time, sample clearance, etc.. We also recommend talking with a music attorney to make sure you are following all of the proper tax guidelines that go along with creating and selling music. 
+                            You need to find out exactly how much you need to raise to create your project. Make sure you discuss all of the costs with your band members, producer, studio, etc… Some unknown costs can come up that are not expected. This can include mixing and master, extra recording time, sample clearance, etc.. We also recommend talking with a music attorney to make sure you are following all of the proper tax guidelines that go along with creating and selling music.
                         </p>
                     </div>
                     <div id="join-3">
                         <h4>How much to give back</h4>
                         <p>
-                            This can be determined by several factory; how big of an artist you are, how much you are trying to raise, and how engaged of an audience you have. If you are a  smaller artist with a small fan base, you might want to give back a bigger percentage to attract more investors. If you are a larger artists with a very engaged fans base you can give back less of a percentage if you anticipate to sell a lot more records.  It's always better to have a smaller percentage of a big pie than to have a big percentage of a small pie. 
+                            This can be determined by several factory; how big of an artist you are, how much you are trying to raise, and how engaged of an audience you have. If you are a  smaller artist with a small fan base, you might want to give back a bigger percentage to attract more investors. If you are a larger artists with a very engaged fans base you can give back less of a percentage if you anticipate to sell a lot more records.  It's always better to have a smaller percentage of a big pie than to have a big percentage of a small pie.
```

Comment:
Yeah, I'm totally re-working this section. Haven't had a chance to update it yet. Some of the content is a little outdated now as well.